src/render_loop_mlx.c

```diff
@@ -0,0 +1,35 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   render_loop_mlx.c                                  :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/19 22:37:57 by gwolf             #+#    #+#             */
+/*   Updated: 2023/12/19 22:54:05 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "render.h"
+
+int	ft_draw_scene(t_render *render)
+{
+	ft_render_image(*render);
+	return (0);
+}
+
+int	ft_end_loop(void *mlx_ptr)
```

Comment:
Sames as above

Reply:
I think I had a problem with passing the function to the loop hook, but it should work. Will check!